tests/suites/test_suite_pkcs11_client.function

```diff
@@ -323,3 +323,194 @@ exit:
     mbedtls_pk_free( &transparent_ctx );
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PK_C:MBEDTLS_SHA256_C */
+void pk_signX_verifyX( int key_type )
```

Comment:
Here and in the next function, the meaning of X and I is not obvious to me. I don't think it's a problem to have long names for the test functions, so `pk_sign_opaque_verify_opaque` would probably be better.